pkg/kubectl/cmd/wait/wait.go

```diff
@@ -67,10 +67,10 @@ var (
 // errNoMatchingResources is returned when there is no resources matching a query.
 var errNoMatchingResources = errors.New("no matching resources found")
 
-// WaitFlags directly reflect the information that CLI is gathering via flags.  They will be converted to Options, which
+// Flags directly reflect the information that CLI is gathering via flags.  They will be converted to Flags, which
 // reflect the runtime requirements for the command.  This structure reduces the transformation to wiring and makes
```

Comment:
Why change `WaitFlags ` to `Flags`?

Reply:
emmm......, can you paster the error in here?

Reply:
IMO, Check where transfer the `wait.WaitFlags`, and rename the import file

Reply:
IMO, named like that seems too easy, How about `CLIWaitFlags`?

Reply:
Not sure. need more opinions here... I used naming convention as followed [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubectl/apply/strategy/merge.go#L22) and [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubelet/server/remotecommand/httpstream.go#L42)